roles/setup_http_store/defaults/main.yml

```diff
@@ -2,6 +2,7 @@ http_store_container_name: http_store
 http_store_pod_name: http_store_pod
 http_dir: /opt/http_store
 http_data_dir: "{{ http_dir }}/data"
+http_port: "{{ hostvars['http_store']['http_store_port'] }}"
```

Comment:
default to 80 here that way we don't make a breaking change to people's inventories net time they pull

Reply:
I’ll do that.

Reply:
+1